websites/D/Discord Rescource/presence.ts

```diff
@@ -0,0 +1,68 @@
+const presence = new Presence({
+		clientId: "973710731549745152"
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "drw",
+			startTimestamp: browsingTimestamp
+		},
+		{ pathname, search } = window.location;
+	switch (true) {
+		case pathname.includes("/wiki"): {
+			presenceData.smallImageKey = "wiki";
+			presenceData.smallImageText = "Wiki";
+			if (pathname === ("/wiki" || "/wiki/"))
```

Comment:
this is not valid javascript, I'm surprised you didn't get TS errors from this

Reply:
> this is not valid javascript, I'm surprised you didn't get TS errors from this

Ohh it's working fine here, do i have to change it or can i leave it like that?